app/main.py

```diff
@@ -1 +1,10 @@
-# write your code here
+def copy_file(command: str) -> None:
+    cmd, source_file, dest_file, *_ = command.split()
```

Comment:
Before unpacking, it would be good to check that the length of the split result equals 3